'make' step fails: libtool:   error: specify a tag with '--tag'

crosstool-NG version: current git 1b18fd6bdd0c4e1b4ade5d0cb02eba5f2d1f78db
host OS: Gentoo Linux, x86_64
host gcc version 6.4.0 (Gentoo 6.4.0-r1 p1.3) 
host libtool (GNU libtool) 2.4.6

I'm doing steps:
```
./bootstrap
./configure --enable-local
make
```

This works fine in crosstool-ng 1.23.0, but no longer in current git.
It fails at the make step, with:
```
/usr/bin/libtool    --mode=link gcc  -g -O2   -o conf conf.o zconf.o  
libtool: link: unable to infer tagged configuration
libtool:   error: specify a tag with '--tag'
gmake[3]: *** [Makefile:496: conf] Error 1
gmake[3]: Leaving directory '/home/tdescham/repo/contrib/crosstool-ng/kconfig'
```

The full log is in attachment.
[crosstool-ng-problem.log](https://github.com/crosstool-ng/crosstool-ng/files/1908602/crosstool-ng-problem.log)

I did a bisect, and following candidates pop up (there was some issue with strip-program that caused me to skip some revisions):

```
There are only 'skip'ped commits left to test.
The first bad commit could be any of:
d401afbddaae56093cce388e62f7ba4d9d68a327
1ce2eab963a004ba352a5cea800e2765a7c6affd
576dcc3fee5b949cbf439fd95ab642dfcfe51a74
0bcb67fa52b1318b098366fa44955f528999f4db
9238b587210c53ba51dd665ffb9ff3e13456880f
7655fc2a3c64fce46517d4a2af047d4c6e663f50
896bc2d17313f32a968efb09917cc3eb2b4e3ed8
66821508f4064904995189f4b46652eb6257b1ea
We cannot bisect more!
```

Most likely candidate seems 9238b587210c53ba51dd665ffb9ff3e13456880f to me, but I could be wrong.

--- a/kconfig/Makefile.am
+++ b/kconfig/Makefile.am
@@ -1,10 +1,8 @@
 ## vim: set noet :
 
-## TBD turn off program renaming for these? Or account for it in ct-ng script?
-## TBD when kconfig is split into a subpackage, need to remove this option from sub-configure
+## FIXME when kconfig is split into a subpackage, need to remove this option from sub-configure
 transform		= s,x,x,
 
-## TBD install into lib/crosstool-ng/kconfig? or move to libexec which seems more suitable
 pkglibexec_PROGRAMS    = conf nconf mconf
 
 EXTRA_DIST		= zconf.y zconf.l \
@@ -17,6 +15,7 @@ BUILT_SOURCES	= zconf.c zconf.lex.c
 AM_LFLAGS		= -L -Pzconf
 AM_YFLAGS		= -l -b zconf -p zconf
 AM_CPPFLAGS		= -include config.h -DCONFIG_=\"CT_\"
+AM_LIBTOOLFLAGS	= --tag CC
 
 conf_SOURCES    = conf.c zconf.c
 

